README.md

```diff
@@ -15,11 +15,37 @@ LPWC also supports irregular time intervals between time points collected in bio
 
 Installation
 ------------
-The first major release of LPWC will be added to CRAN.
-Until then, the easiest way to install LPWC is as follows:
+Prior to analyzing your data, the R package needs to be installed.
+
+The easiest way to install LPWC is through R:
+
 ``` r
+install.packages("LPWC")
+``
+
+There are other additional ways to download LPWC.
+The first option is most useful if using a previous versions of LPWC
```

Comment:
`version`

Does this mean that the first option is best if they already have a previous version of LPWC and want to update to the current version?